scripts/setup.sh

```diff
@@ -36,11 +41,46 @@ OKTA_REGISTRY=${ARTIFACTORY_URL}/api/npm/npm-okta-master
 echo "Replacing $YARN_REGISTRY with $OKTA_REGISTRY within yarn.lock files..."
 sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" yarn.lock
 
-if ! yarn install ; then
+# Install dependencies but do not build
+if ! yarn install --frozen-lockfile --ignore-scripts; then
   echo "yarn install failed! Exiting..."
   exit ${FAILED_SETUP}
 fi
 
 # Revert the original change(s)
 echo "Replacing $OKTA_REGISTRY with $YARN_REGISTRY within yarn.lock files..."
 sed -i "s#${OKTA_REGISTRY}#${YARN_REGISTRY}#" yarn.lock
+
+# Install a specific version of auth-js, used by downstream artifact builds
+if [ ! -z "$AUTHJS_VERSION" ]; then
+  echo "Installing AUTHJS_VERSION: ${AUTHJS_VERSION}"
+  npm config set strict-ssl false
+
+  AUTHJS_URI=https://artifacts.aue1d.saasure.com/artifactory/npm-topic/@okta/okta-auth-js/-/@okta/okta-auth-js-${AUTHJS_VERSION}.tgz
+  if ! yarn add -DW --ignore-scripts ${AUTHJS_URI}; then
```

Comment:
recommend including the --force flag. Sometimes on major version boundaries it will fail to install